[package] arrow/1.0.0: doesn't compile with_snappy=True

<!-- 
  Please don't forget to update the issue title.
  Include all applicable information to help us reproduce your problem.
-->

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **arrow/1.0.0**
  * Operating System+version: **Linux Ubuntu 18.04**
  * Compiler+version: **GCC 8.4.0**
  * Conan version: **conan 1.28.1**
  * Python version: **Python 3.6.9**


### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
Configuration for profile default:

[settings]
os=Linux
os_build=Linux
arch=x86_64
arch_build=x86_64
compiler=gcc
compiler.version=8
compiler.libcxx=libstdc++11
build_type=Release
[options]
[build_requires]
[env]
```


### Steps to reproduce (Include if Applicable)
Trying to add arrow dependency with_snappy support, in my conanfile.txt:
```
[requires]
arrow/1.0.0

[generators]
cmake

[options]
arrow:shared=True
arrow:parquet=True
arrow:with_zlib=True
arrow:with_snappy=True

[imports]
include,* -> ./include
lib,* -> ./lib
bin,* -> ./bin

```
Snappy cannot be found by cmake:
```
CMake Error in source_subfolder/cpp/src/arrow/CMakeLists.txt:
  Found relative path while evaluating include directories of "arrow_objlib":

    "SNAPPY_INCLUDE_DIRS-NOTFOUND"
```

### Logs (Include/Attach if Applicable)
<details><summary>Click to expand log</summary>

cmake logs:
```
-- Found Boost: /usr/include (found suitable version "1.65.1", minimum required is "1.58") found components: program_options filesystem system
--
-- -------------------- BUILD SUMMARY --------------------
--
--   Installation dir          : /home/ms/dev/test_project/docker/install
--   Build type                : Debug
--   Build tests               : ON
--   Boost include dirs        : /usr/include
--
-- ------------------------------------------------------
-- Found Boost: /usr/include (found suitable version "1.65.1", minimum required is "1.65") found components: system filesystem fiber serialization context thread chrono date_time atomic
-- Conan: checking conan executable
-- Conan: Found program /usr/local/bin/conan
-- Conan: Version found Conan version 1.28.1

-- Conan: Automatic detection of conan settings from cmake
-- Conan: Settings= -s;build_type=Release;-s;compiler=gcc;-s;compiler.version=8;-s;compiler.libcxx=libstdc++11
-- Conan: checking conan executable
-- Conan: Found program /usr/local/bin/conan
-- Conan: Version found Conan version 1.28.1

-- Conan executing: /usr/local/bin/conan install /home/ms/dev/test_project/conanfile.txt -s build_type=Release -s compiler=gcc -s compiler.version=8 -s compiler.libcxx=libstdc++11 -g=cmake --build=missing
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++11
compiler.version=8
os=Linux
os_build=Linux
[options]
[build_requires]
[env]

conanfile.txt: Installing package
Requirements
    arrow/1.0.0 from 'conan-center' - Cache
    boost/1.73.0 from 'conan-center' - Cache
    bzip2/1.0.8 from 'conan-center' - Cache
    gtest/1.10.0 from 'conan-center' - Cache
    openssl/1.1.1g from 'conan-center' - Cache
    snappy/1.1.8 from 'conan-center' - Cache
    thrift/0.13.0 from 'conan-center' - Cache
    zlib/1.2.11 from 'conan-center' - Cache
Packages
    arrow/1.0.0:43bbcaeeb1a802972a5b4bcd43a59965ce0a452a - Build
    boost/1.73.0:7068ed10403246f8331cfec6288fde7b48f7bc8a - Cache
    bzip2/1.0.8:a5875aed3fc7ae8dd0488f7e5e99acbc480d721d - Cache
    glm/0.9.9.8:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    gtest/1.10.0:319986a8ddc0fc392032c63f81cacfe9edd93e70 - Cache
    libevent/2.1.11:cf55fc10a66c8837ed109a38d536e5514904c418 - Cache
    openssl/1.1.1g:1d877a3df840030e6a8abb74c5ffb9088d08b47a - Cache
    protobuf/3.6.1@bincrafters/stable:56e0cf6d16ee57367a0661ab743f4e43b29223f8 - Cache
    snappy/1.1.8:56e0cf6d16ee57367a0661ab743f4e43b29223f8 - Cache
    thrift/0.13.0:6dd2415c8070244fde33c3d37b2b6f98d355e7a8 - Cache
    zlib/1.2.11:1d877a3df840030e6a8abb74c5ffb9088d08b47a - Cache

Installing (downloading, building) binaries...
bzip2/1.0.8: Already installed!
glm/0.9.9.8: Already installed!
gtest/1.10.0: Already installed!
openssl/1.1.1g: Already installed!
protobuf/3.6.1@bincrafters/stable: Already installed!
snappy/1.1.8: Already installed!
zlib/1.2.11: Already installed!
boost/1.73.0: Already installed!
boost/1.73.0: LIBRARIES: ['boost_wave', 'boost_container', 'boost_contract', 'boost_exception', 'boost_graph', 'boost_iostreams', 'boost_locale', 'boost_log', 'boost_program_options', 'boost_random', 'boost_regex', 'boost_serialization', 'boost_wserialization', 'boost_coroutine', 'boost_fiber', 'boost_context', 'boost_timer', 'boost_thread', 'boost_chrono', 'boost_date_time', 'boost_atomic', 'boost_filesystem', 'boost_system', 'boost_type_erasure', 'boost_log_setup', 'boost_math_c99', 'boost_math_c99f', 'boost_math_c99l', 'boost_math_tr1', 'boost_math_tr1f', 'boost_math_tr1l', 'boost_nowide', 'boost_stacktrace_addr2line', 'boost_stacktrace_backtrace', 'boost_stacktrace_basic', 'boost_stacktrace_noop', 'boost_unit_test_framework']
boost/1.73.0: Package folder: /home/ms/.conan/data/boost/1.73.0/_/_/package/7068ed10403246f8331cfec6288fde7b48f7bc8a
libevent/2.1.11: Already installed!
thrift/0.13.0: Already installed!
thrift/0.13.0: Appending PATH env var with : /home/ms/.conan/data/thrift/0.13.0/_/_/package/6dd2415c8070244fde33c3d37b2b6f98d355e7a8/bin
arrow/1.0.0: Copying sources to build folder
arrow/1.0.0: Building your package in /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a
arrow/1.0.0: Generator cmake_find_package created Findthrift.cmake
arrow/1.0.0: Generator cmake_find_package created FindSnappy.cmake
arrow/1.0.0: Generator cmake_find_package created FindBoost.cmake
arrow/1.0.0: Generator cmake_find_package created FindLibevent.cmake
arrow/1.0.0: Generator cmake_find_package created FindZLIB.cmake
arrow/1.0.0: Generator cmake_find_package created FindBZip2.cmake
arrow/1.0.0: Generator cmake_find_package created FindOpenSSL.cmake
arrow/1.0.0: Generator cmake created conanbuildinfo.cmake
arrow/1.0.0: Calling build()
-- The C compiler identification is GNU 8.4.0
-- The CXX compiler identification is GNU 8.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: called by CMake conan helper
-- Conan: called inside local cache
-- Conan: Adjusting output directories
-- Conan: Using cmake global configuration
-- Conan: Adjusting default RPATHs Conan policies
-- Conan: Adjusting language standard
-- Conan: Compiler GCC>=5, checking major version 8
-- Conan: Checking correct version: 8
-- Conan: C++ stdlib: libstdc++11
-- Building using CMake version: 3.17.2
-- Arrow version: 1.0.0 (full: '1.0.0')
-- Arrow SO version: 100 (full: 100.0.0)
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1")
-- clang-tidy not found
-- clang-format not found
-- Could NOT find ClangTools (missing: CLANG_FORMAT_BIN CLANG_TIDY_BIN)
-- infer not found
fatal: not a git repository (or any parent up to mount point /home/ms)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
-- Found Python3: /usr/bin/python3.6 (found version "3.6.9") found components: Interpreter
-- Using ccache: /usr/local/bin/ccache
-- Found cpplint executable at CPPLINT_BIN-NOTFOUND
-- System processor: x86_64
-- Performing Test CXX_SUPPORTS_SSE4_2
-- Performing Test CXX_SUPPORTS_SSE4_2 - Success
-- Performing Test CXX_SUPPORTS_AVX2
-- Performing Test CXX_SUPPORTS_AVX2 - Success
-- Performing Test CXX_SUPPORTS_AVX512
-- Performing Test CXX_SUPPORTS_AVX512 - Success
-- Arrow build warning level: PRODUCTION
Using ld linker
Configured for RELEASE build (set with cmake -DCMAKE_BUILD_TYPE={release,debug,...})
-- Build Type: RELEASE
-- Using SYSTEM approach to find dependencies
-- ARROW_ABSL_BUILD_VERSION: 2eba343b51e0923cd3fb919a6abd6120590fc059
-- ARROW_AWSSDK_BUILD_VERSION: 1.7.160
-- ARROW_BOOST_BUILD_VERSION: 1.71.0
-- ARROW_BROTLI_BUILD_VERSION: v1.0.7
-- ARROW_BZIP2_BUILD_VERSION: 1.0.8
-- ARROW_CARES_BUILD_VERSION: 1.16.1
-- ARROW_GBENCHMARK_BUILD_VERSION: v1.5.1
-- ARROW_GFLAGS_BUILD_VERSION: v2.2.2
-- ARROW_GLOG_BUILD_VERSION: v0.4.0
-- ARROW_GRPC_BUILD_VERSION: v1.29.1
-- ARROW_GTEST_BUILD_VERSION: 1.8.1
-- ARROW_JEMALLOC_BUILD_VERSION: 5.2.1
-- ARROW_LZ4_BUILD_VERSION: v1.9.2
-- ARROW_MIMALLOC_BUILD_VERSION: v1.6.3
-- ARROW_ORC_BUILD_VERSION: 1.6.2
-- ARROW_PROTOBUF_BUILD_VERSION: v3.12.1
-- ARROW_RAPIDJSON_BUILD_VERSION: 1a803826f1197b5e30703afe4b9c0e7dd48074f5
-- ARROW_RE2_BUILD_VERSION: 2019-08-01
-- ARROW_SNAPPY_BUILD_VERSION: 1.1.8
-- ARROW_THRIFT_BUILD_VERSION: 0.12.0
-- ARROW_THRIFT_BUILD_MD5_CHECKSUM: 3deebbb4d1ca77dd9c9e009a1ea02183
-- ARROW_UTF8PROC_BUILD_VERSION: v2.5.0
-- ARROW_ZLIB_BUILD_VERSION: 1.2.11
-- ARROW_ZSTD_BUILD_VERSION: v1.4.5
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE
-- Conan: Using autogenerated FindSnappy.cmake
-- Found Snappy: 1.1.8 (found version "1.1.8")
-- Library snappy found /home/ms/.conan/data/snappy/1.1.8/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/lib/libsnappy.a
-- Found: /home/ms/.conan/data/snappy/1.1.8/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/lib/libsnappy.a
-- Library snappy found /home/ms/.conan/data/snappy/1.1.8/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/lib/libsnappy.a
-- Found: /home/ms/.conan/data/snappy/1.1.8/_/_/package/56e0cf6d16ee57367a0661ab743f4e43b29223f8/lib/libsnappy.a
-- Building without OpenSSL support. Minimum OpenSSL version 1.0.2 required.
-- Checking for module 'thrift'
--   No package 'thrift' found
-- Found Thrift: /home/ms/.conan/data/thrift/0.13.0/_/_/package/6dd2415c8070244fde33c3d37b2b6f98d355e7a8/lib/libthrift.a (found suitable version "0.13.0", minimum required is "0.11.0")
-- Conan: Using autogenerated FindZLIB.cmake
-- Found ZLIB: 1.2.11 (found version "1.2.11")
-- Library z found /home/ms/.conan/data/zlib/1.2.11/_/_/package/1d877a3df840030e6a8abb74c5ffb9088d08b47a/lib/libz.a
-- Found: /home/ms/.conan/data/zlib/1.2.11/_/_/package/1d877a3df840030e6a8abb74c5ffb9088d08b47a/lib/libz.a
-- Found hdfs.h at: /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/thirdparty/hadoop/include/hdfs.h
-- All bundled static libraries:
-- CMAKE_C_FLAGS:   -m64 -O3 -DNDEBUG  -Wall -msse4.2
-- CMAKE_CXX_FLAGS:  -Wno-noexcept-type   -m64 -fdiagnostics-color=always -O3 -DNDEBUG  -Wall -msse4.2
-- ---------------------------------------------------------------------
-- Arrow version:                                 1.0.0
--
-- Build configuration summary:
--   Generator: Unix Makefiles
--   Build type: RELEASE
--   Source directory: /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
--   Install prefix: /home/ms/.conan/data/arrow/1.0.0/_/_/package/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a
--
-- Compile and link options:
--
--   ARROW_CXXFLAGS="" [default=""]
--       Compiler flags to append when compiling Arrow
--   ARROW_BUILD_STATIC=False [default=ON]
--       Build static libraries
--   ARROW_BUILD_SHARED=True [default=ON]
--       Build shared libraries
--   ARROW_PACKAGE_KIND="" [default=""]
--       Arbitrary string that identifies the kind of package
--       (for informational purposes)
--   ARROW_GIT_ID="" [default=""]
--       The Arrow git commit id (if any)
--   ARROW_GIT_DESCRIPTION="" [default=""]
--       The Arrow git commit description (if any)
--   ARROW_NO_DEPRECATED_API=False [default=OFF]
--       Exclude deprecated APIs from build
--   ARROW_USE_CCACHE=ON [default=ON]
--       Use ccache when compiling (if available)
--   ARROW_USE_LD_GOLD=OFF [default=OFF]
--       Use ld.gold for linking on Linux (if available)
--   ARROW_USE_PRECOMPILED_HEADERS=OFF [default=OFF]
--       Use precompiled headers when compiling
--   ARROW_SIMD_LEVEL=SSE4_2 [default=NONE|SSE4_2|AVX2|AVX512]
--       SIMD compiler optimization level
--   ARROW_ARMV8_ARCH=armv8-a [default=armv8-a|armv8-a+crc+crypto]
--       Arm64 arch and extensions
--   ARROW_ALTIVEC=ON [default=ON]
--       Build with Altivec if compiler has support
--   ARROW_RPATH_ORIGIN=OFF [default=OFF]
--       Build Arrow libraries with RATH set to $ORIGIN
--   ARROW_INSTALL_NAME_RPATH=False [default=ON]
--       Build Arrow libraries with install_name set to @rpath
--   ARROW_GGDB_DEBUG=ON [default=ON]
--       Pass -ggdb flag to debug builds
--
-- Test and benchmark options:
--
--   ARROW_BUILD_EXAMPLES=False [default=OFF]
--       Build the Arrow examples
--   ARROW_BUILD_TESTS=False [default=OFF]
--       Build the Arrow googletest unit tests
--   ARROW_ENABLE_TIMING_TESTS=False [default=ON]
--       Enable timing-sensitive tests
--   ARROW_BUILD_INTEGRATION=False [default=OFF]
--       Build the Arrow integration test executables
--   ARROW_BUILD_BENCHMARKS=False [default=OFF]
--       Build the Arrow micro benchmarks
--   ARROW_BUILD_BENCHMARKS_REFERENCE=OFF [default=OFF]
--       Build the Arrow micro reference benchmarks
--   ARROW_TEST_LINKAGE=shared [default=shared|static]
--       Linkage of Arrow libraries with unit tests executables.
--   ARROW_FUZZING=OFF [default=OFF]
--       Build Arrow Fuzzing executables
--   ARROW_LARGE_MEMORY_TESTS=OFF [default=OFF]
--       Enable unit tests which use large memory
--
-- Lint options:
--
--   ARROW_ONLY_LINT=OFF [default=OFF]
--       Only define the lint and check-format targets
--   ARROW_VERBOSE_LINT=OFF [default=OFF]
--       If off, 'quiet' flags will be passed to linting tools
--   ARROW_GENERATE_COVERAGE=OFF [default=OFF]
--       Build with C++ code coverage enabled
--
-- Checks options:
--
--   ARROW_TEST_MEMCHECK=OFF [default=OFF]
--       Run the test suite using valgrind --tool=memcheck
--   ARROW_USE_ASAN=OFF [default=OFF]
--       Enable Address Sanitizer checks
--   ARROW_USE_TSAN=OFF [default=OFF]
--       Enable Thread Sanitizer checks
--   ARROW_USE_UBSAN=OFF [default=OFF]
--       Enable Undefined Behavior sanitizer checks
--
-- Project component options:
--
--   ARROW_BUILD_UTILITIES=False [default=OFF]
--       Build Arrow commandline utilities
--   ARROW_COMPUTE=ON [default=OFF]
--       Build the Arrow Compute Modules
--   ARROW_CSV=False [default=OFF]
--       Build the Arrow CSV Parser Module
--   ARROW_CUDA=False [default=OFF]
--       Build the Arrow CUDA extensions (requires CUDA toolkit)
--   ARROW_DATASET=False [default=OFF]
--       Build the Arrow Dataset Modules
--   ARROW_FILESYSTEM=False [default=OFF]
--       Build the Arrow Filesystem Layer
--   ARROW_FLIGHT=False [default=OFF]
--       Build the Arrow Flight RPC System (requires GRPC, Protocol Buffers)
--   ARROW_GANDIVA=False [default=OFF]
--       Build the Gandiva libraries
--   ARROW_HDFS=False [default=OFF]
--       Build the Arrow HDFS bridge
--   ARROW_HIVESERVER2=False [default=OFF]
--       Build the HiveServer2 client and Arrow adapter
--   ARROW_IPC=ON [default=ON]
--       Build the Arrow IPC extensions
--   ARROW_JEMALLOC=False [default=ON]
--       Build the Arrow jemalloc-based allocator
--   ARROW_JNI=OFF [default=OFF]
--       Build the Arrow JNI lib
--   ARROW_JSON=False [default=OFF]
--       Build Arrow with JSON support (requires RapidJSON)
--   ARROW_MIMALLOC=OFF [default=OFF]
--       Build the Arrow mimalloc-based allocator
--   ARROW_PARQUET=True [default=OFF]
--       Build the Parquet libraries
--   ARROW_ORC=OFF [default=OFF]
--       Build the Arrow ORC adapter
--   ARROW_PLASMA=False [default=OFF]
--       Build the plasma object store along with Arrow
--   ARROW_PLASMA_JAVA_CLIENT=OFF [default=OFF]
--       Build the plasma object store java client
--   ARROW_PYTHON=OFF [default=OFF]
--       Build the Arrow CPython extensions
--   ARROW_S3=False [default=OFF]
--       Build Arrow with S3 support (requires the AWS SDK for C++)
--   ARROW_TENSORFLOW=OFF [default=OFF]
--       Build Arrow with TensorFlow support enabled
--
-- Thirdparty toolchain options:
--
--   ARROW_DEPENDENCY_SOURCE=SYSTEM [default=AUTO|BUNDLED|SYSTEM|CONDA|BREW]
--       Method to use for acquiring arrow's build dependencies
--   ARROW_VERBOSE_THIRDPARTY_BUILD=True [default=OFF]
--       Show output from ExternalProjects rather than just logging to files
--   ARROW_BOOST_USE_SHARED=ON [default=ON]
--       Rely on boost shared libraries where relevant
--   ARROW_BROTLI_USE_SHARED=False [default=ON]
--       Rely on Brotli shared libraries where relevant
--   ARROW_GFLAGS_USE_SHARED=ON [default=ON]
--       Rely on GFlags shared libraries where relevant
--   ARROW_GRPC_USE_SHARED=ON [default=ON]
--       Rely on gRPC shared libraries where relevant
--   ARROW_PROTOBUF_USE_SHARED=ON [default=ON]
--       Rely on Protocol Buffers shared libraries where relevant
--   ARROW_ZSTD_USE_SHARED=ON [default=ON]
--       Rely on zstd shared libraries where relevant
--   ARROW_WITH_BACKTRACE=False [default=ON]
--       Build with backtrace support
--   ARROW_USE_GLOG=False [default=OFF]
--       Build libraries with glog support for pluggable logging
--   ARROW_WITH_BROTLI=False [default=OFF]
--       Build with Brotli compression
--   ARROW_WITH_BZ2=False [default=OFF]
--       Build with BZ2 compression
--   ARROW_WITH_LZ4=False [default=OFF]
--       Build with lz4 compression
--   ARROW_WITH_SNAPPY=True [default=OFF]
--       Build with Snappy compression
--   ARROW_WITH_ZLIB=True [default=OFF]
--       Build with zlib compression
--   ARROW_WITH_ZSTD=False [default=OFF]
--       Build with zstd compression
--   ARROW_WITH_UTF8PROC=False [default=ON]
--       Build with support for Unicode properties using the utf8proc library
--
-- Parquet options:
--
--   PARQUET_MINIMAL_DEPENDENCY=OFF [default=OFF]
--       Depend only on Thirdparty headers to build libparquet.
--       Always OFF if building binaries
--   PARQUET_BUILD_EXECUTABLES=OFF [default=OFF]
--       Build the Parquet executable CLI tools. Requires static libraries to be built.
--   PARQUET_BUILD_EXAMPLES=OFF [default=OFF]
--       Build the Parquet examples. Requires static libraries to be built.
--   PARQUET_REQUIRE_ENCRYPTION=False [default=OFF]
--       Build support for encryption. Fail if OpenSSL is not found
--
-- Gandiva options:
--
--   ARROW_GANDIVA_JAVA=OFF [default=OFF]
--       Build the Gandiva JNI wrappers
--   ARROW_GANDIVA_STATIC_LIBSTDCPP=OFF [default=OFF]
--       Include -static-libstdc++ -static-libgcc when linking with
--       Gandiva static libraries
--   ARROW_GANDIVA_PC_CXX_FLAGS="" [default=""]
--       Compiler flags to append when pre-compiling Gandiva operations
--
-- Advanced developer options:
--
--   ARROW_EXTRA_ERROR_CONTEXT=OFF [default=OFF]
--       Compile with extra error context (line numbers, code)
--   ARROW_OPTIONAL_INSTALL=OFF [default=OFF]
--       If enabled install ONLY targets that have already been built. Please be
--       advised that if this is enabled 'install' will fail silently on components
--       that have not been built
--   Outputting build configuration summary to /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/cmake_summary.json
-- Configuring done
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
SNAPPY_INCLUDE_DIRS
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/testing
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/testing
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/testing
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/testing
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/testing
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/testing
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/array
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/array
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/array
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/array
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/array
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/array
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/c
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/c
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/c
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/c
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/c
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/c
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/io
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/io
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/io
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/io
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/io
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/io
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/tensor
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/tensor
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/tensor
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/tensor
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/tensor
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/tensor
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/util
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/util
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/util
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/util
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/util
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/util
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored/datetime
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored/datetime
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored/datetime
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored/datetime
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored/datetime
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored/datetime
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored/double-conversion
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored/double-conversion
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored/double-conversion
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored/double-conversion
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored/double-conversion
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/vendored/double-conversion
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/compute
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/compute
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/compute
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/compute
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/compute
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/compute
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/compute
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/compute/kernels
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/compute/kernels
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/compute/kernels
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/compute/kernels
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/compute/kernels
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/compute/kernels
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/ipc
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/ipc
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/ipc
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/ipc
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/ipc
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/ipc
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/arrow/ipc
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet/api
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet/api
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet/api
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet/api
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet/api
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet/api
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/src/parquet/arrow
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/tools/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/tools/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/tools/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/tools/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/tools/parquet
   used as include directory in directory /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a/source_subfolder/cpp/tools/parquet

CMake Error in source_subfolder/cpp/src/arrow/CMakeLists.txt:
  Found relative path while evaluating include directories of "arrow_objlib":

    "SNAPPY_INCLUDE_DIRS-NOTFOUND"



CMake Error in source_subfolder/cpp/src/arrow/CMakeLists.txt:
  Found relative path while evaluating include directories of "arrow_objlib":

    "SNAPPY_INCLUDE_DIRS-NOTFOUND"



CMake Error in source_subfolder/cpp/src/arrow/CMakeLists.txt:
  Found relative path while evaluating include directories of "arrow_objlib":

    "SNAPPY_INCLUDE_DIRS-NOTFOUND"



CMake Error in source_subfolder/cpp/src/arrow/CMakeLists.txt:
  Found relative path while evaluating include directories of "arrow_objlib":

    "SNAPPY_INCLUDE_DIRS-NOTFOUND"



CMake Error in source_subfolder/cpp/src/arrow/CMakeLists.txt:
  Found relative path while evaluating include directories of "arrow_shared":

    "SNAPPY_INCLUDE_DIRS-NOTFOUND"



CMake Error in source_subfolder/cpp/src/arrow/CMakeLists.txt:
  Found relative path while evaluating include directories of "arrow_shared":

    "SNAPPY_INCLUDE_DIRS-NOTFOUND"



CMake Error in source_subfolder/cpp/src/parquet/CMakeLists.txt:
  Found relative path while evaluating include directories of
  "parquet_shared":

    "SNAPPY_INCLUDE_DIRS-NOTFOUND"



CMake Error in source_subfolder/cpp/src/parquet/CMakeLists.txt:
  Found relative path while evaluating include directories of
  "parquet_objlib":

    "SNAPPY_INCLUDE_DIRS-NOTFOUND"



CMake Error in source_subfolder/cpp/src/parquet/CMakeLists.txt:
  Found relative path while evaluating include directories of
  "parquet_objlib":

    "SNAPPY_INCLUDE_DIRS-NOTFOUND"



-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
arrow/1.0.0:
arrow/1.0.0: WARN: Package is corrupted, removing folder: /home/ms/.conan/data/arrow/1.0.0/_/_/package/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a
arrow/1.0.0: WARN: Build folder is dirty, removing it: /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a
arrow/1.0.0: ERROR: Package '43bbcaeeb1a802972a5b4bcd43a59965ce0a452a' build failed
arrow/1.0.0: WARN: Build folder /home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a
ERROR: arrow/1.0.0: Error in build() method, line 351
	cmake = self._configure_cmake()
while calling '_configure_cmake', line 338
	self._cmake.configure()
	ConanException: Error 1 while executing cd '/home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a' && cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE="Release" -DCONAN_IN_LOCAL_CACHE="ON" -DCONAN_COMPILER="gcc" -DCONAN_COMPILER_VERSION="8" -DCONAN_CXX_FLAGS="-m64" -DCONAN_SHARED_LINKER_FLAGS="-m64" -DCONAN_C_FLAGS="-m64" -DCONAN_LIBCXX="libstdc++11" -DBUILD_SHARED_LIBS="ON" -DCMAKE_INSTALL_PREFIX="/home/ms/.conan/data/arrow/1.0.0/_/_/package/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a" -DCMAKE_INSTALL_BINDIR="bin" -DCMAKE_INSTALL_SBINDIR="bin" -DCMAKE_INSTALL_LIBEXECDIR="bin" -DCMAKE_INSTALL_LIBDIR="lib" -DCMAKE_INSTALL_INCLUDEDIR="include" -DCMAKE_INSTALL_OLDINCLUDEDIR="include" -DCMAKE_INSTALL_DATAROOTDIR="share" -DCMAKE_MODULE_PATH="/home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a" -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY="ON" -DCONAN_EXPORTED="1" -DARROW_DEPENDENCY_SOURCE="SYSTEM" -DARROW_GANDIVA="False" -DARROW_PARQUET="True" -DARROW_PLASMA="False" -DARROW_DATASET="False" -DARROW_FILESYSTEM="False" -DPARQUET_REQUIRE_ENCRYPTION="False" -DARROW_HDFS="False" -DARROW_VERBOSE_THIRDPARTY_BUILD="True" -DARROW_BUILD_SHARED="True" -DARROW_BUILD_STATIC="False" -DARROW_NO_DEPRECATED_API="False" -DARROW_FLIGHT="False" -DARROW_HIVESERVER2="False" -DARROW_COMPUTE="False" -DARROW_CSV="False" -DARROW_CUDA="False" -DARROW_JEMALLOC="False" -DARROW_JSON="False" -DBOOST_SOURCE="SYSTEM" -DProtobuf_SOURCE="SYSTEM" -DgRPC_SOURCE="SYSTEM" -DARROW_USE_GLOG="False" -DGLOG_SOURCE="SYSTEM" -DARROW_WITH_BACKTRACE="False" -DARROW_WITH_BROTLI="False" -DBrotli_SOURCE="SYSTEM" -DARROW_BROTLI_USE_SHARED="False" -Dgflags_SOURCE="SYSTEM" -DARROW_WITH_BZ2="False" -DBZip2_SOURCE="SYSTEM" -DARROW_WITH_LZ4="False" -DLz4_SOURCE="SYSTEM" -DARROW_WITH_SNAPPY="True" -DSnappy_SOURCE="SYSTEM" -DARROW_WITH_ZLIB="True" -DRE2_SOURCE="SYSTEM" -DZLIB_SOURCE="SYSTEM" -DARROW_WITH_ZSTD="False" -DZSTD_SOURCE="SYSTEM" -DORC_SOURCE="SYSTEM" -DARROW_WITH_THRIFT="True" -DThrift_SOURCE="SYSTEM" -DTHRIFT_VERSION="1.0" -DARROW_USE_OPENSSL="False" -DARROW_S3="False" -DAWSSDK_SOURCE="SYSTEM" -DARROW_BUILD_UTILITIES="False" -DARROW_BUILD_INTEGRATION="False" -DARROW_INSTALL_NAME_RPATH="False" -DARROW_BUILD_EXAMPLES="False" -DARROW_BUILD_TESTS="False" -DARROW_ENABLE_TIMING_TESTS="False" -DARROW_BUILD_BENCHMARKS="False" -DLLVM_SOURCE="SYSTEM" -DARROW_WITH_UTF8PROC="False" -Dutf8proc_SOURCE="SYSTEM" -Wno-dev '/home/ms/.conan/data/arrow/1.0.0/_/_/build/43bbcaeeb1a802972a5b4bcd43a59965ce0a452a'
CMake Error at build/conan.cmake:402 (message):
  Conan install failed='1'
Call Stack (most recent call first):
  build/conan.cmake:497 (conan_cmake_install)
  CMakeLists.txt:101 (conan_cmake_run)


-- Configuring incomplete, errors occurred!
See also "/home/ms/dev/test_project/build/CMakeFiles/CMakeOutput.log".
See also "/home/ms/dev/test_project/build/CMakeFiles/CMakeError.log".
```


</details>

--- a/recipes/arrow/all/patches/1.0.0-0001-cmake.patch
+++ b/recipes/arrow/all/patches/1.0.0-0001-cmake.patch
@@ -20,7 +20,7 @@
      if(NOT Snappy_FOUND)
        find_package(SnappyAlt)
      endif()
-@@ -866,14 +866,14 @@ if(ARROW_WITH_SNAPPY)
+@@ -866,7 +866,7 @@
    elseif(Snappy_SOURCE STREQUAL "SYSTEM")
      # SnappyConfig.cmake is not installed on Ubuntu/Debian
      # TODO: Make a bug report upstream
@@ -29,15 +29,6 @@
      if(NOT Snappy_FOUND)
        find_package(SnappyAlt REQUIRED)
      endif()
-   endif()
- 
-   # TODO: Don't use global includes but rather target_include_directories
--  get_target_property(SNAPPY_INCLUDE_DIRS Snappy::snappy INTERFACE_INCLUDE_DIRECTORIES)
--  include_directories(SYSTEM ${SNAPPY_INCLUDE_DIRS})
-+  #get_target_property(SNAPPY_INCLUDE_DIRS Snappy::snappy INTERFACE_INCLUDE_DIRECTORIES)
-+  include_directories(${CONAN_INCLUDE_DIRS_SNAPPY})
- endif()
- 
 @@ -938,7 +938,7 @@ endmacro()
  if(ARROW_WITH_BROTLI)
    resolve_dependency(Brotli)

--- a/recipes/arrow/all/patches/1.0.0-0001-cmake.patch
+++ b/recipes/arrow/all/patches/1.0.0-0001-cmake.patch
@@ -1,36 +1,3 @@
---- cpp/CMakeLists.txt
-+++ cpp/CMakeLists.txt
-@@ -654,7 +654,7 @@ endif()
- 
- if(ARROW_WITH_BROTLI)
-   # Order is important for static linking
--  set(ARROW_BROTLI_LIBS Brotli::brotlienc Brotli::brotlidec Brotli::brotlicommon)
-+  set(ARROW_BROTLI_LIBS Brotli::Brotli)
-   list(APPEND ARROW_LINK_LIBS ${ARROW_BROTLI_LIBS})
-   list(APPEND ARROW_STATIC_LINK_LIBS ${ARROW_BROTLI_LIBS})
- endif()
-@@ -668,8 +668,8 @@ if(ARROW_WITH_LZ4)
- endif()
- 
- if(ARROW_WITH_SNAPPY)
--  list(APPEND ARROW_STATIC_LINK_LIBS Snappy::snappy)
-+  list(APPEND ARROW_STATIC_LINK_LIBS Snappy::Snappy)
- endif()
- 
- if(ARROW_WITH_ZLIB)
-   list(APPEND ARROW_STATIC_LINK_LIBS ZLIB::ZLIB)
-@@ -685,9 +685,9 @@ if(ARROW_ORC)
- endif()
- 
- if(ARROW_USE_GLOG)
--  list(APPEND ARROW_LINK_LIBS glog::glog)
--  list(APPEND ARROW_STATIC_LINK_LIBS glog::glog)
-+  list(APPEND ARROW_LINK_LIBS GLOG::GLOG)
-+  list(APPEND ARROW_STATIC_LINK_LIBS GLOG::GLOG)
-   add_definitions("-DARROW_USE_GLOG")
- endif()
- 
- if(ARROW_S3)
 --- cpp/cmake_modules/DefineOptions.cmake
 +++ cpp/cmake_modules/DefineOptions.cmake
 @@ -76,7 +76,7 @@ macro(define_option_string name description default)
@@ -66,8 +33,9 @@
  
    # TODO: Don't use global includes but rather target_include_directories
 -  get_target_property(SNAPPY_INCLUDE_DIRS Snappy::snappy INTERFACE_INCLUDE_DIRECTORIES)
-+  get_target_property(SNAPPY_INCLUDE_DIRS Snappy::Snappy INTERFACE_INCLUDE_DIRECTORIES)
-   include_directories(SYSTEM ${SNAPPY_INCLUDE_DIRS})
+-  include_directories(SYSTEM ${SNAPPY_INCLUDE_DIRS})
++  #get_target_property(SNAPPY_INCLUDE_DIRS Snappy::snappy INTERFACE_INCLUDE_DIRECTORIES)
++  include_directories(${CONAN_INCLUDE_DIRS_SNAPPY})
  endif()
  
 @@ -938,7 +938,7 @@ endmacro()
@@ -75,19 +43,12 @@
    resolve_dependency(Brotli)
    # TODO: Don't use global includes but rather target_include_directories
 -  get_target_property(BROTLI_INCLUDE_DIR Brotli::brotlicommon
-+  get_target_property(BROTLI_INCLUDE_DIR Brotli::Brotli
-                       INTERFACE_INCLUDE_DIRECTORIES)
-   include_directories(SYSTEM ${BROTLI_INCLUDE_DIR})
+-                      INTERFACE_INCLUDE_DIRECTORIES)
+-  include_directories(SYSTEM ${BROTLI_INCLUDE_DIR})
++  #get_target_property(BROTLI_INCLUDE_DIR Brotli::brotlicommon
++  #                   INTERFACE_INCLUDE_DIRECTORIES)
++  include_directories(SYSTEM ${CONAN_INCLUDE_DIRS_BROTLI})
  endif()
-@@ -1057,7 +1057,7 @@ endmacro()
- if(ARROW_USE_GLOG)
-   resolve_dependency(GLOG)
-   # TODO: Don't use global includes but rather target_include_directories
--  get_target_property(GLOG_INCLUDE_DIR glog::glog INTERFACE_INCLUDE_DIRECTORIES)
-+  get_target_property(GLOG_INCLUDE_DIR GLOG::GLOG INTERFACE_INCLUDE_DIRECTORIES)
-   include_directories(SYSTEM ${GLOG_INCLUDE_DIR})
- endif()
- 
 @@ -1139,8 +1139,8 @@
      build_gflags()
    elseif(gflags_SOURCE STREQUAL "SYSTEM")

--- a/recipes/arrow/all/patches/1.0.0-0001-cmake.patch
+++ b/recipes/arrow/all/patches/1.0.0-0001-cmake.patch
@@ -29,17 +29,6 @@
      if(NOT Snappy_FOUND)
        find_package(SnappyAlt REQUIRED)
      endif()
-@@ -938,7 +938,7 @@ endmacro()
- if(ARROW_WITH_BROTLI)
-   resolve_dependency(Brotli)
-   # TODO: Don't use global includes but rather target_include_directories
--  get_target_property(BROTLI_INCLUDE_DIR Brotli::brotlicommon
--                      INTERFACE_INCLUDE_DIRECTORIES)
--  include_directories(SYSTEM ${BROTLI_INCLUDE_DIR})
-+  #get_target_property(BROTLI_INCLUDE_DIR Brotli::brotlicommon
-+  #                   INTERFACE_INCLUDE_DIRECTORIES)
-+  include_directories(SYSTEM ${CONAN_INCLUDE_DIRS_BROTLI})
- endif()
 @@ -1139,8 +1139,8 @@
      build_gflags()
    elseif(gflags_SOURCE STREQUAL "SYSTEM")

